internal/controller/serverset/serverset_test.go

```diff
@@ -1349,116 +1349,113 @@ func Test_serverSetController_updateOrRecreateVolumes_activeReplicaUpdatedLast_c
 	assertions.Equal("nic1-0-0-0", nicController.lastMethodCall[deleteMethod][secondArg])
 }
 
-func Test_serverSetController_Delete(t *testing.T) {
```

Comment:
is this test temporarily removed ?

Reply:
Probably permanently. It wasn't really testing much